Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WPF/Interop/WinRT/BindingExpression.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
+{
+    /// <summary>
+    /// <see cref="Windows.UI.Xaml.Data.BindingExpression"/>
+    /// </summary>
+    public class BindingExpression
+    {
+        internal Windows.UI.Xaml.Data.BindingExpression UwpInstance { get; }
```

Comment:
I didn't check all instances, but those I did check only had internal (within the class itself) usage. Can we convert this to a private field? In cases where I needed access to the WinRT types I did what I needed to in a class that encapsulated all that logic, then exposed the simpler projections